sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -664,19 +669,68 @@ func tagImage(ctx context.Context, imageName string, newImageName string, client
 
 // withDynamicResolver provides an initialized resolver for use with ref.
 func withDynamicResolver(ctx context.Context, ref string) containerd.RemoteOpt {
-	if !strings.HasPrefix(ref, "ecr.aws/") {
-		// not handled here
-		return func(_ *containerd.Client, _ *containerd.RemoteContext) error { return nil }
-	}
-
-	return func(_ *containerd.Client, c *containerd.RemoteContext) error {
-		// Create the ECR resolver
-		resolver, err := ecr.NewResolver()
+	noOpt := func(_ *containerd.Client, _ *containerd.RemoteContext) error { return nil }
```

Comment:
*nit*

Was this name supposed to be "no-op" or no opt? No-op seems more apt in this case.

Reply:
I was originally going for no-option, but I think you're right that no op is more befitting in this case.